src/python_be.cc

```diff
@@ -2038,7 +2040,48 @@ TRITONBACKEND_Initialize(TRITONBACKEND_Backend* backend)
   TRITONBACKEND_ArtifactType artifact_type;
   RETURN_IF_ERROR(
       TRITONBACKEND_BackendArtifacts(backend, &artifact_type, &location));
-  backend_state->python_lib = location;
+
+  // Check if `triton_python_backend_stub` and `triton_python_backend_utils.py`
+  // are located under `location`.
+  std::string default_python_backend_dir =
```

Comment:
Add [TODO] for forward slash to be platform agnostic (i.e., for Windows we need to use backward slash).

cc @fpetrini15

Reply:
Added.